### DISCUSSION TITLE
How to move the position of components and layouts?

### DISCUSSION THREAD

**Discussion by Seikaijyu** [AllenDang/giu#532] [Q&A] [answered]
I want to move the position of the widget or layout, preferably relative to the position of the widget. What should I do?

**Comment by gucio321** [comment, marked as answer]
The only way i see it's to use SetCursorPosition to set position of drawing cursor

**Reply by Seikaijyu** [reply]
Oh, this name makes me think it is to set the mouse position

**Reply by Seikaijyu** [reply]
I don't quite understand what cursor stands for here

**Reply by gucio321** [reply]
Yeah cursor stands from drawing cursor that draws widgets

**Reply by Seikaijyu** [reply]
I see. Thank you for your answer
